apps/src/appMain.js

```diff
@@ -42,6 +44,24 @@ module.exports = function(app, levels, options) {
           options.level.levelBuilderRecommendedBlocks);
     }
 
+    if (marked && options.level.authoredHints) {
+      var hints;
+      try {
+        hints = JSON.parse(options.level.authoredHints);
+      } catch (e) {
+        hints = [];
+      }
+      level.authoredHints = hints.map(function (hint) {
+        return {
+          content: marked(hint.hint_markdown),
+          hintId: hint.hint_id,
+          hintClass: hint.hint_class,
+          hintType: hint.hint_type,
+          alreadySeen: false
+        };
+      });
+    }
```

Comment:
Oh yeah, it totally should.

And yes; it's similar in pattern to the "options.level.levelbuilderRequiredBlocks" that become "level.requiredBlocks", but by overwriting the variable we don't have the unnecessary "levelbuilderAuthoredHints" value just hanging out for the rest of the session